Vision/samples/V1/SamplegenNoConfig.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
ℹ️ **Note**: this "no config" sample is different from the rest. Rather than demonstrating a feature, it shows what a sample looks like when _not configured at all_.

These samples _could_ be generated for every API method without any manual configuration.

Review of the "no config" sample is not required, this is more of an FYI :)